deps/include/mpv/client.h

```diff
@@ -1824,7 +1824,7 @@ MPV_EXPORT int mpv_hook_add(mpv_handle *ctx, uint64_t reply_userdata,
  * Respond to a MPV_EVENT_HOOK event. You must call this after you have handled
  * the event. There is no way to "cancel" or "stop" the hook.
  *
- * Calling this will will typically unblock the player for whatever the hook
+ * Calling this will typically unblock the player for whatever the hook
```

Comment:
Files under deps/include can not be fixed/changed by IINA. These source files belong to the [mpv](https://github.com/mpv-player/mpv) and [FFmpeg](https://ffmpeg.org/) projects. Whenever the IINA project updates the versions of the libraries it uses from these projects the include files corresponding to the version of the library are obtained from those projects and cached in the IINA git repository. This can be seen in the [history](https://github.com/iina/iina/commits/develop/deps/include/mpv/client.h) for client.h.

The typo is still present in the mpv sources as seen [here](https://github.com/mpv-player/mpv/blob/11ed012c343e32da1c7a0cb8ddb7006f53f1f0bc/libmpv/client.h#L1827).

To fix client.h a pull request needs to be posted [here](https://github.com/mpv-player/mpv/pulls) to the [mpv](https://github.com/mpv-player/mpv) project with just the fix to client.h.